app/controllers/monthly_bills_controller.rb

```diff
@@ -2,9 +2,9 @@ class MonthlyBillsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @authorized_user = MonthlyBill.joins(:user).where(["monthly_bills.user_id = ? OR monthly_bills.user_id = ? OR users.invited_by_id = ?",
+    authorized_user = MonthlyBill.joins(:user).where(["monthly_bills.user_id = ? OR monthly_bills.user_id = ? OR users.invited_by_id = ?",
                                                        current_user.id, current_user.invited_by_id, current_user.id])
-    @search = @authorized_user.order(due_date: 'ASC').ransack(params[:q])
+    @search = authorized_user.order(due_date: 'ASC').ransack(params[:q])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.